src/Apps/Settings/Routes/MyCollection/Components/EmptyMyCollectionPage.tsx

```diff
@@ -1,23 +1,17 @@
 import { Join, Spacer } from "@artsy/palette"
-import { useSystemContext } from "System"
 import { useFeatureFlag } from "System/useFeatureFlag"
 import { HowMyCollectionWorks } from "./HowMyCollectionWorks"
 import { MyCollectionAppDownload } from "./MyCollectionAppDownload"
 import { MyCollectionBenefits } from "./MyCollectionBenefits"
 import { MyCollectionEmptyState } from "./MyCollectionEmptyState"
 
-export interface EmptyMyCollectionPageProps {
-  loggedOutState?: boolean
-}
-
 export const EmptyMyCollectionPage: React.FC = () => {
   const enableMyCollectionPhase3 = useFeatureFlag("my-collection-web-phase-3")
-  const { isLoggedIn } = useSystemContext()
 
   return (
-    <Join separator={<Spacer y={6} />}>
+    <Join separator={<Spacer y={[6, 12]} />}>
       {enableMyCollectionPhase3 ? (
-        <MyCollectionEmptyState loggedOutState={!isLoggedIn} />
```

Comment:
We don't need to thread this through and if we did `loggedOutState` is a confusing prop name.